action/governance/withdrawFunds.go

```diff
@@ -99,37 +99,45 @@ func runWithdraw(ctx *action.Context, signedTx action.RawTx) (bool, action.Respo
 		}
 		return false, result
 	}
+
 	fundStore := ctx.ProposalMasterStore.ProposalFund
 	currentFundsForProposal := fundStore.GetCurrentFundsForProposal(proposal.ProposalID)
-	// if funding goal is reached or there is still time for funding
-	if currentFundsForProposal.BigInt().Cmp(proposal.FundingGoal.BigInt()) >= 0 || ctx.Header.Height <= proposal.FundingDeadline {
-		ctx.Logger.Error("Proposal does not meet withdraw requirement", withdrawProposal.ProposalID)
-		result := action.Response{
-			Events: action.GetEvent(withdrawProposal.Tags(), "withdraw_proposal_does_not_meet_withdraw_requirement"),
-			Log:    governance.ErrProposalWithdrawNotEligible.Marshal(),
+	// if outcome is not cancelled or insufficient funds
+	ctx.Logger.Debug("outcome :", proposal.Outcome)
```

Comment:
Detail might be better

Reply:
oh yes

Reply:
actually since this is only for debugging, I'll remove it.